src/styles/app.module.css

```diff
@@ -2,14 +2,38 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Montserrat, 'Times New Roman', Times, serif;
+}
+
+nav {
+  padding: 1.3rem;
```

Comment:
- [ ] Kindly ensure that your app has the same spacing between elements as the given design and that the missing designs highlighted below are added.

<img width="1231" alt="Screenshot 2023-06-15 at 22 20 24" src="https://github.com/forLoop94/book-store/assets/64774936/2d53821f-da52-4ee7-841e-f9e35af9dc5d">